libs/blocks/section-metadata/section-metadata.css

```diff
@@ -133,20 +137,118 @@ main>.section[class*='-up']>.content {
   gap: var(--spacing-xxxl);
 }
 
-.section.two-up,
-.section.three-up,
-.section.four-up,
-.section.five-up {
+.section[class*='-up'].no-col-gap {
+  column-gap: 0;
+}
+
+.section[class*='-up'].xxs-col-gap {
+  column-gap: var(--spacing-xxs);
+}
+
+.section[class*='-up'].xs-col-gap {
+  column-gap: var(--spacing-xs);
+}
+
+.section[class*='-up'].s-col-gap {
+  column-gap: var(--spacing-s);
+}
+
+.section[class*='-up'].l-col-gap {
+  column-gap: var(--spacing-l);
+}
+
+.section[class*='-up'].xl-col-gap {
+  column-gap: var(--spacing-xl)
+}
+
+.section[class*='-up'].xxl-col-gap {
+  column-gap: var(--spacing-xxl);
+}
+
+.section[class*='-up'].xxxl-col-gap {
+  column-gap: var(--spacing-xxxl);
+}
+
+.section[class*='-up'].no-row-gap {
+  row-gap: 0;
+}
+
+.section[class*='-up'].xxs-row-gap {
+  row-gap: var(--spacing-xxs);
+}
+
+.section[class*='-up'].xs-row-gap {
+  row-gap: var(--spacing-xs);
+}
+
+.section[class*='-up'].s-row-gap {
+  row-gap: var(--spacing-s);
+}
+
+.section[class*='-up'].l-row-gap {
+  row-gap: var(--spacing-l);
+}
+
+.section[class*='-up'].xl-row-gap {
+  row-gap: var(--spacing-xl)
+}
+
+.section[class*='-up'].xxl-row-gap {
+  row-gap: var(--spacing-xxl);
+}
+
+.section[class*='-up'].xxxl-row-gap {
+  row-gap: var(--spacing-xxxl);
+}
+
+.section[class*='-up'] {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(276px, 1fr));
   gap: var(--spacing-m);
+  grid-template-columns: repeat(auto-fit, minmax(276px, 1fr));
   align-items: start;
   padding-left: var(--grid-margins-width);
   padding-right: var(--grid-margins-width);
 }
 
-.section.five-up {
-  grid-template-columns: repeat(auto-fit, minmax(142px, 1fr));
+.section.auto-up {
+  grid-template-columns: repeat(var(--section-grid-columns), 1fr);
+}
+
+.section.no-padding {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.section.two-up-mobile {
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.section.three-up-mobile {
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.section.four-up-mobile {
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.section.five-up-mobile {
+  grid-template-columns: repeat(5, 1fr);
+}
```

Comment:
These are the same as the `.section.x-up` entries (lines 321-335).  Can these be moved to there?  Or is there even a need for a `three-up-mobile` if it's the same as `three-up`?  I don't see any other uses of `three-up-mobile`.

Same applies to the x-up-tablet classes.